src/rgw/radosgw-admin/radosgw-admin.hpp

```diff
@@ -0,0 +1,89 @@
+#ifndef RADOSGW_ADMIN_HPP 
```

Comment:
Were there already large arguments about whether to use this or #pragma once? Let's go with this, if not...

Reply:
I don't actually think there was an argument. I get the impression the consensus favors `#pragma once` but I don't think it favors it particularly *strongly*.

Reply:
+1. I'll leave it as-is for now then. I have a /slight/ preference for the include-guard, until Modules-- however, it's pretty weak and I am totally fine changing it if anyone feels otherwise.